server/util/map.js

```diff
@@ -157,6 +158,54 @@ async function getTrafficByLocation(coordinates) {
 	return traffic;
 }
 
+async function createCrime(crimeList) {
+	const createdCrimes = [];
+
+	for (const crime of crimeList) {
+		const createdCrime = new Crime({
+			type: crime.type,
+			address: crime.address,
+			date: crime.date,
+			location: {
+				coordinates: [crime.location.longtitude, crime.location.latitude],
+			},
+		});
+
+		try {
+			const sess = await mongoose.startSession();
+			sess.startTransaction();
+			await createdCrime.save({ session: sess });
+			await sess.commitTransaction();
+			createdCrimes.push(createdCrime);
+		} catch (err) {
+			console.error(err);
+		}
+	}
+
+	return createdCrimes;
+}
+
+async function getCrimeByLocation(coordinates) {
+	let crime;
+	await Crime.collection.createIndex({ location: '2dsphere' });
```

Comment:
Hmmm why are we creating index here? Shouldn't index be added to the model/schema file instead?